src/__tests__/date.test.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-ignore TS6133
+import { expect, test } from "@jest/globals";
+
+import * as z from "../index";
+
+const benchmarkDate = new Date(2022, 10, 5);
+
+const minCheck = z.date().min(benchmarkDate);
+const maxCheck = z.date().max(benchmarkDate);
+
+test("passing validations", () => {
+  minCheck.parse(new Date(benchmarkDate));
+  minCheck.parse(new Date(benchmarkDate.getTime() + 1));
+
+  maxCheck.parse(new Date(benchmarkDate));
+  maxCheck.parse(new Date(benchmarkDate.getTime() - 1));
```

Comment:
I would feel safer if you added an equality check as well. Both these should pass, if the check is inclusive:

```ts
minCheck.parse(new Date(benchmarkDate.getTime())

maxCheck.parse(new Date(benchmarkDate.getTime())
```

Reply:
No, wait, sorry, keep reading things wrong. That's what the previous line without the `.getTime()` does 🤦‍♂️

Nevermind me!